src/com/company/Secondary.java

```diff
@@ -2,6 +2,6 @@
 
 public class Secondary {
     public static void main(String[] args) {
-        System.out.println("Hello, general Grivus");
+        System.out.println("Hello, general Grivus!");
```

Comment:
could you please change general to General

Reply:
Yep